website-profile.html

```diff
@@ -1 +1,54 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Lewis's Profile</title>
+<link rel="stylesheet" href="style.css">
+<body>
+
+<div class="top-banner">
+    <img class="lewis-ski-img" src="lewis-ski-img.jpeg" alt="Lewis on the ski slopes." style="float:right;width:300px;height:400px;">
```

Comment:
Separation of concerns. When we separate our HTML and CSS into separate files, it's usually better to stick to that separation throughout the code, rather than having some inline CSS and some imported CSS.